actions/templates/delete/index.js

```diff
@@ -70,7 +75,7 @@ async function main(params) {
       };
     }
     const fullTemplateName = (orgName !== undefined) ? orgName + '/' + templateName : templateName;
-    const template = await findTemplateByName(fullTemplateName, params.TEMPLATE_REGISTRY_ORG, params.TEMPLATE_REGISTRY_REPOSITORY);
+    const template = await findTemplateByName(dbParams, fullTemplateName);
     if (null === template) {
       return {
         'statusCode': 404
```

Comment:
We also need to remove checking the registry.json file below, new templates won't be there